src/main/kotlin/org/rust/lang/core/macros/Include.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.lang.core.macros
+
+import org.rust.lang.core.psi.RsLitExpr
+import org.rust.lang.core.psi.RsMacroCall
+import org.rust.lang.core.psi.RsMacroExpr
+import org.rust.lang.core.psi.RsPsiFactory
+import org.rust.lang.core.psi.ext.stringLiteralValue
+import org.rust.lang.core.resolve.DEFAULT_RECURSION_LIMIT
+import java.nio.ByteBuffer
+
+fun expandInclude(call: RsMacroCall): List<RsExpandedElement>? {
+    val psiFactory = RsPsiFactory(call.project)
+    val args = call.includeMacroArgument?.expr ?: return null
+    val path = expandStringArgument(args) ?: return null
+    val file = call.containingFile.virtualFile.parent.findFileByRelativePath(path) ?: return null
```

Comment:
I think we can access PSI directly without re-parsing: `file.toPsiFile(call.project)`.
(don't sure about it)

Reply:
Hmm, my proposal will work only if the following conditions are met:
1. context is not RsMacroExpr/RsPatMacro/RsMacroType
2. included file has ".rs" extension

So you solution seems to be more full